catalog/plugin/catalog-plugin-query-monitor-commands/src/main/java/ddf.catalog.querymonitorcommands/QueryMonitorCommandRemoveFromTable.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.querymonitorcommands;
+
+import org.apache.karaf.shell.api.action.Action;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.querymonitor.QueryMonitorPlugin;
+
+@Command(scope = "catalog", name = "setremovefromtable", description = "Sets flag to determine whether to delete active searches from table once they complete")
+
+public class QueryMonitorCommandRemoveFromTable implements Action {
+    @Argument(name = "trueOrFalse", description = "The boolean value indicating whether to remove activeSearches from the search table after the search has completed.", index = 0, multiValued = false, required = true)
```

Comment:
The argument could probably use a better name than `trueOrFalse`. Maybe something like `removeAfterComplete`?